src/jupyter_contrib_nbextensions/nbextensions/zenmode/main.js

```diff
@@ -13,12 +13,10 @@ define([
     "require",
     "jquery",
     "base/js/namespace",
-    "base/js/events"
 ], function(
     require,
     $,
     IPython,
-    events
```

Comment:
The events object is needed in new line 107, so this message to go back in. Everything else looks good :)